file_app/management/commands/load_files.py

```diff
@@ -0,0 +1,65 @@
+from django.core.management.base import BaseCommand
+from file_app.models import FileData
+from django.core import exceptions
+import os
+import csv
+
+
+# ========================================== Command Class inheriting BaseCommand ==============================
+class Command(BaseCommand):
+    help = "Loading files to the database"
+
+    """ this function handling the all operations"""
+    def handle(self, *args, **options):
+
+        """ here you have to set you directory(folder) path of you machine """
+        path = 'C:\\Users\\Muhammad Noman\\Desktop\\weatherman-master\\weatherfiles'
+        files = []
+
+        """ this loop getting all the files from path(directory) and storing into files list """
+        for r, d, f in os.walk(path):
+            for file in f:
+                if '.txt' in file:
+                    files.append(os.path.join(r, file))
+
+        count = 0
+        """ this loop getting file name from files list and opening it to load data from the file """
+        for f in files:
+            count = count + 1
+            print(count)
```

Comment:
while printing the count ? if you really need progress of your command while its running in background the try logging the data. see [logging](https://docs.djangoproject.com/en/2.2/topics/logging/) topic and configure some custom loggers.